packages/eslint-plugin/src/rules/no-inline-notification-imports-from-component-library.ts

```diff
@@ -0,0 +1,251 @@
+import { Rule } from "eslint"
+import getCompleteObjectKeyRange from "../util/getCompleteObjectKeyRange"
+
+// TODO - transform this rule into fixing all legacy packages, not just InlineNotification
+// TODO - kaizen version awareness
+
+module.exports = {
+  meta: {
+    type: "problem",
+    docs: {
+      description:
+        "Fix breaking changes from @kaizen >=11.0.0. InlineNotification component moved from @kaizen/component-library to @kaizen/notification. See more at https://github.com/cultureamp/kaizen-design-system/blob/master/packages/component-library/CHANGELOG.md#1100-2021-08-30",
```

Comment:
How hard would it be to expand this to cover the other notification components as well?
* GlobalNotification
* ToastNotification
* addToastNotification
* clearToastNotifications
* removeToastNotification

Reply:
Should be easy, just need sometime to make it generic, seems like we need to address those TODO comments from the get go!